c2corg_ui/static/js/utils.js

```diff
@@ -123,6 +123,30 @@ app.utils.formatDate = function(date) {
 
 
 /**
+ * check that the start date if before end date.
+ * Returns an error message if there is an error with dates.
+ * @export
+ * @param {Object} outing
+ * @return {(undefined | string)}
+ */
+app.utils.checkDates = function(outing) {
+  var start = new Date(outing['date_start']);
+  var end = new Date(outing['date_end']);
+
+  if (!end) {
+    return;
+  }
+  if (start > end) {
+    return 'The starting date is after the ending date.';
```

Comment:
I don't think it's too easy to detect it in .js files, how would the extractor know which are translatable or not?
I will add those to i18n file.